### PR TITLE
Add FAQ to about page

### PR DESCRIPTION
**Pull request by imphil**
@wallento @olofk + everybody else: please have a look at the FAQ text to make sure the wording is ok before it goes online.

### CODE REVIEW

File: site/sitecontent/about.md [hunk 1]
```diff
@@ -9,3 +9,39 @@ Towards this goal, LibreCores provides you
...
+## Frequently Asked Questions (FAQ)
+
+### Who's running LibreCores?
+LibreCores is a project of the [Free and Open Source Silicon Foundation](http://www.fossi-foundation.org) (FOSSi), which was created to give a voice to the digital hardware design community. LibreCores, like FOSSi, is run entirely by volunteers.
+
+### How does LibreCores relate to OpenCores?
+LibreCores advances the idea of OpenCores.org to give the community a place to share projects, ideas, and knowledge in the area of free and open source digital hardware design.
+
+As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our power to evolve the concept together with the creators of OpenCores. Unfortunately, to this day, we were not yet able to achieve this goal.
+Therefore, we were forced to use the LibreCores brand instead for the time being.
```
Comment (wallento) [inline site/sitecontent/about.md]: I would formulate slightly different: "Therefore, we were forced to use the LibreCores brand instead for the time being." -> "Therefore, we decided to give it a real fresh start under the LibreCores brand."

File: site/sitecontent/about.md [hunk 2]
```diff
@@ -9,3 +9,39 @@ Towards this goal, LibreCores provides you
...
 
+## Frequently Asked Questions (FAQ)
+
+### Who's running LibreCores?
+LibreCores is a project of the [Free and Open Source Silicon Foundation](http://www.fossi-foundation.org) (FOSSi), which was created to give a voice to the digital hardware design community. LibreCores, like FOSSi, is run entirely by volunteers.
+
+### How does LibreCores relate to OpenCores?
+LibreCores advances the idea of OpenCores.org to give the community a place to share projects, ideas, and knowledge in the area of free and open source digital hardware design.
+
+As heavy users of OpenCores, we (the FOSSi Foundation) tried everything in our power to evolve the concept together with the creators of OpenCores. Unfortunately, to this day, we were not yet able to achieve this goal.
```
Comment (wallento) [inline site/sitecontent/about.md]: creators of OpenCores -> owners of OpenCores

File: site/sitecontent/about.md [hunk 3]
```diff
@@ -9,3 +9,39 @@ Towards this goal, LibreCores provides you
...
+Therefore, we were forced to use the LibreCores brand instead for the time being.
+
+### What are the future plans for LibreCores?
+At the moment, LibreCores is in a "public beta" state.
```
Comment (wallento) [inline site/sitecontent/about.md]: maybe alpha for the moment (no list, no search, no org)?

**Comment by imphil** [comment]
thanks @wallento I've updated the PR with your comments.

**Comment by olofk** [comment]
You got my blessing to push this

**Comment by oleg-nenashev** [comment]
Thanks for fixing it! I was planning to create an issue for it, but you were fixed it much quicker :)

**Pull request merged**